### ISSUE TITLE
Speech recognition seems to ignore the word `procent` in combination with a number.

### ISSUE THREAD

**Issue by OpenMindNL** [athombv/homey#2290]
**Metadata**

* **Homey Version**: 1.5.7
* **Homey Log Code**:  does not apply
* **Smartphone Platform**: does not apply
* **Smartphone App Version**: does not apply

**If related to a Homey App:**
* **Homey App Version**: does not apply
* **Homey App Log Code**: does not apply
* **Affected Devices (preferably product codes)**: does not apply

**What did you try to achieve?**
Make Homey understand a spoken command with `100 procent` in it.

**What did you expect as result?**
To see:  `Homey: 100 procent` at `http://homey-ip/manager/speech-input/`

**What was the result?**

`Homey: 100` at `http://homey-ip/manager/speech-input/`

**Any other remarks**
When I say: `OK Homey... procent` (without number), it shows
`Homey: procent` at `http://homey-ip/manager/speech-input/`

**Comment by xAPPO** [comment]
It’s actually percent in English.    ‘Purr’ ‘scent’ ... don’t know if that makes any difference ?

**Comment by OpenMindNL** [comment]
@xAPPO I'm aware of the english percent... but I'm running my Homey in dutch, so I am not sure if `percent` is ignored also. That's why I mentioned my dutch `procent` experience.

**Comment by xAPPO** [comment]
Yeah sorry.. was just trying to help. I saw this on GitHub where there’s no language mention or background and everything is in English. Since later reading the Slack post I have seen the background.

**Comment by OpenMindNL** [comment]
@xAPPO No problem at all! I actually only thought that the people of Athom would understand. I have not thought about it at all that it could be confusing for English readers. :)
